test/net/sourceforge/kolmafia/preferences/PreferencesTest.java

```diff
@@ -488,12 +522,12 @@ public void exerciseIsPerUserGlobalProperty() {
   @Test
   public void actuallySaveFileToIncreaseCoverage() {
```

Comment:
In general, I would prefer if tests actually assert something about the resulting state, rather than just calling a bunch of methods with side effects.

Coverage that doesn't check anything about the code only tells you that the happy path doesn't cause crashes, not that the code is correct.

(ditto below with exerciseReset)

Reply:
I agree in general.  But a test that covers a change is better than a change with no coverage.  It is not clear but this test did pass before the change to Preferences as well as after.  I'm open to suggestions as to how to test something that is not visible, produces only side effects and verifying the side effect is irrelevant to the code that was changed.  I can ponder this a little harder.

Reply:
Not static.  No new variable.